### ISSUE TITLE
Opening modal forces all content to the left.

### ISSUE THREAD

**Issue by HarkonenBade** [twbs/bootstrap#17399]
Opening a modal forces all content to the left.
Tracked this down to line 2061 of bootstrap.js:
``` javascript
document.body.style.paddingRight = bodyPadding + (this._scrollbarWidth + 'px');
```
Which evaluates to a padding of 1516px on my desktop (rather than the 31px it should be).
I believe this is due to mis-bracketing.

The source line that controls this appears to be ~~[bootstrap.js:L1180](https://github.com/twbs/bootstrap/blob/master/dist/js/bootstrap.js#L1180)~~

**Comment by cvrebert** [comment]
Please mention which version of Bootstrap you're using next time.
And you linked to the wrong version (`master` is currently v3).
Here's the correct link: https://github.com/twbs/bootstrap/blob/8941bdfbda237bed621935cac439520eddc79150/dist/js/bootstrap.js#L2061

**Comment by HarkonenBade** [comment]
Sorry for the lack of mentioning the version.

I think I've found the original source line that corresponds to the fault: [modal.js:L429](https://github.com/twbs/bootstrap/blob/v4-dev/js/src/modal.js#L429)

I believe that those two lines should instead be:
``` javascript
document.body.style.paddingRight =
          `${bodyPadding + this._scrollbarWidth}px`
```

**Comment by cvrebert** [comment]
Fixed by #17457.
